test/e2e/policyrecommendation_test.go

```diff
@@ -237,6 +251,8 @@ func testPolicyRecommendationFailed(t *testing.T, data *TestData) {
 	})
 	require.NoError(t, err)
 	driverPodName := fmt.Sprintf("%s-driver", jobName)
+	err = data.podWaitForReady(defaultTimeout, jobName+"-driver", flowVisibilityNamespace)
```

Comment:
`driverPodName` has been declared in the previous line.

Reply:
FIxed. Thanks